src/Sulu/Bundle/ContentBundle/Resources/views/Preview/script.html.twig

```diff
@@ -37,7 +38,7 @@
                 init: function() {
                     try {
                         // create websocket
-                        this.conn = new WebSocket(this.url + ':' + this.port);
+                        this.conn = new WebSocket(this.url + ':' + this.port + this.route);
```

Comment:
no because in the old preview there is no dependencies ... but this will be removed in a new PR soon!